components/courses/app/policies/courses/interview/approve_policy.rb

```diff
@@ -0,0 +1,26 @@
+module Courses
+  class Interview < ApplicationRecord
+    class ApprovePolicy
+      def initialize(user, season, interview)
+        @mentor = ::Courses::Mentor.find_by(user_id: user.id, season_id: season.id)
+        @interview = interview
+      end
+
+      def allowed?
+        current_user_interviewer? && interview_completed?
+      end
+
+      private
+
+      attr_reader :mentor, :interview
+
+      def current_user_interviewer?
+        interview.mentor == mentor
+      end
+
+      def interview_completed?
+        interview.completed?
```

Comment:
redundant method, because it only swaps `.` to `_`. Use just `interview.completed?`